### PR TITLE
Make airdrop badge value dynamically growing

### PR DESCRIPTION
**Pull request by oskarvu**
Closes FRO-707

https://github.com/marsfoundation/spark-app/assets/43062492/f549924c-4492-417d-adb3-d7e259776cff

Note: I noticed accordion story regression (I thought it was fixed after updating to storybook 8, but it happened again). I turned off accordion open state on mobile/tablet to prevent false regression reporting for this story in future.

### CODE REVIEW

File: packages/app/src/domain/wallet/airdropInfo.ts [hunk 1]
```diff
@@ -4,17 +4,20 @@ import { z } from 'zod'
 import { blockAnaliticaApiUrl } from '@/config/consts'
 import { normalizedUnitNumberSchema } from '../common/validation'
 import { CheckedAddress } from '../types/CheckedAddress'
-import { NormalizedUnitNumber } from '../types/NumericValues'
 
 const airdropInfoResponseSchema = z
   .object({
     token_reward_total: normalizedUnitNumberSchema,
+    token_rate: normalizedUnitNumberSchema,
```
Comment (krzkaczor) [inline packages/app/src/domain/wallet/airdropInfo.ts:11]: can we leave a comment here explaining what token_rate is exactly? is it per second? per minute? alternatively, just rename it to `tokenRatePerX`

File: packages/app/src/features/navbar/logic/use-airdrop-info/useGrowingAirdropAmount.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { NormalizedUnitNumber } from '@/domain/types/NumericValues'
+import { useEffect, useState } from 'react'
+import { Airdrop } from '../../types'
+
+export function useGrowingAirdropAmount(airdrop: Airdrop): NormalizedUnitNumber {
```
[the remark this answers is not in the document]
Comment (krzkaczor) [reply packages/app/src/features/navbar/logic/use-airdrop-info/useGrowingAirdropAmount.ts:5]: yeah i think what @yivlad suggested makes sense but additionally, i think it shouldn't happen all the time IMO. When user is not hovering either the freq of updates should drop significantly (1 per second? 10 seconds?) or simply do not update when is not hovered upon.

File: packages/app/src/domain/wallet/airdropInfo.ts [hunk 3]
```diff
@@ -4,17 +4,20 @@ import { z } from 'zod'
...
   .transform((o) => ({
     tokenReward: o.token_reward_total,
+    tokenRatePerSecond: o.token_rate,
+    airdropTimestamp: o.timestamp,
   }))
   // @note: Api is returning empty object for addresses without airdrop
-  .or(z.object({}).transform(() => ({ tokenReward: NormalizedUnitNumber(0) })))
+  .or(z.object({}).transform(() => null))
 
 export type AirdropInfoResponse = z.infer<typeof airdropInfoResponseSchema>
```
Comment (krzkaczor) [inline packages/app/src/domain/wallet/airdropInfo.ts:22]: btw. perhaps this should be moved into `features/navbar/logic/use-airdrop-info`.

File: packages/app/src/features/navbar/logic/use-airdrop-info/useGrowingAirdropAmount.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (oskarvu) [reply packages/app/src/features/navbar/logic/use-airdrop-info/useGrowingAirdropAmount.ts:5]: Working on it :)

File: packages/app/src/ui/atoms/accordion/Accordion.stories.tsx [hunk 4]
```diff
@@ -56,5 +56,5 @@ export const SingleOpenable: Story = {
   play: async ({ canvasElement }) => (await within(canvasElement).findByText('Item number one')).click(),
 }
 
-export const Mobile = getMobileStory(MultipleOpenable)
-export const Tablet = getTabletStory(MultipleOpenable)
+export const Mobile: Story = { ...getMobileStory(MultipleOpenable), play: undefined }
```
Comment (krzkaczor) [inline packages/app/src/ui/atoms/accordion/Accordion.stories.tsx:59]: please leave a comment that play is disabled to aid vis reg. Alternatively, simply diable these stories for vis reg.

File: packages/app/src/features/navbar/logic/use-airdrop-info/useGrowingAirdropAmount.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import { NormalizedUnitNumber } from '@/domain/types/NumericValues'
+import { useTimestamp } from '@/utils/useTimestamp'
+import { Airdrop } from '../../types'
+import { adjustTokenReward } from './utils/adjustTokenReward'
+
+export function useGrowingAirdropAmount(airdrop: Airdrop, enabled: boolean): NormalizedUnitNumber {
+  const { timestampInMs } = useTimestamp({
+    refreshIntervalInMs: enabled ? airdrop.refreshIntervalInMs : 0,
```
Comment (krzkaczor) [inline packages/app/src/features/navbar/logic/use-airdrop-info/useGrowingAirdropAmount.ts:8]: shouldn't it be undefined instaed of 0? I hope it doesn't cause constant updates 😆

**Review by krzkaczor — APPROVED** [review]

**Review by yivlad — APPROVED** [review]

**Pull request merged**